CommonCompilerConfig.cmake

```diff
@@ -170,20 +158,13 @@ if(MSVC)
 							$<$<CONFIG:Release>:/O2 /Oi /Gy  /GL /MD> 
 							$<$<CONFIG:Debug>:/Ob0 /Od /Zi /sdl /RTC1 /MDd>)
 	#linker flags
-	if("${BITNESS}" STREQUAL "32")
-		set(machine "/MACHINE:X86")
-	else()
-		set(machine "/MACHINE:X${BITNESS}")
-	endif()
 	set(INTERNAL_LINKER_FLAGS "${machine} /NOLOGO /SAFESEH:NO /NODEFAULTLIB:LIBCMT /ignore:4099 /LTCG")
 
 	set(COMMON_COMPILER_DEFINITIONS 	
 		_CONSOLE
 		_MBCS
 		NTA_OS_WINDOWS
 		NTA_COMPILER_MSVC
-		NTA_INTERNAL=1
-		NTA_ARCH_${BITNESS}
```

Comment:
was able to remove NTA_INTERNAL, NTA_ARCH_bitness

Reply:
👍